Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/Flask-0.12.2.dist-info'...
Listing './lib/python3.5/site-packages/Flask_Cache-0.13.1.egg-info'...
Listing './lib/python3.5/site-packages/Jinja2-2.10.dist-info'...
Listing './lib/python3.5/site-packages/LatLon'...
Compiling './lib/python3.5/site-packages/LatLon/lat_lon.py'...
***   File "./lib/python3.5/site-packages/LatLon/lat_lon.py", line 580
    raise NameError, 'Class GeoVector requires two arguments (dx and dy or initial_heading and distance)'
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/LatLon-1.0.2.egg-info'...
Listing './lib/python3.5/site-packages/MarkupSafe-1.0.egg-info'...
Listing './lib/python3.5/site-packages/Shapely-1.6.3.dist-info'...
Listing './lib/python3.5/site-packages/Werkzeug-0.14.1.dist-info'...
Listing './lib/python3.5/site-packages/certifi'...
Listing './lib/python3.5/site-packages/certifi-2017.11.5.dist-info'...
Listing './lib/python3.5/site-packages/chardet'...
Listing './lib/python3.5/site-packages/chardet/cli'...
Listing './lib/python3.5/site-packages/chardet-3.0.4.dist-info'...
Listing './lib/python3.5/site-packages/click'...
Listing './lib/python3.5/site-packages/click-6.7.dist-info'...
Listing './lib/python3.5/site-packages/concurrent'...
Listing './lib/python3.5/site-packages/concurrent/futures'...
Compiling './lib/python3.5/site-packages/concurrent/futures/_base.py'...
***   File "./lib/python3.5/site-packages/concurrent/futures/_base.py", line 381
    raise exception_type, self._exception, self._traceback
                        ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/flask'...
Listing './lib/python3.5/site-packages/flask/ext'...
Listing './lib/python3.5/site-packages/flask_cache'...
Listing './lib/python3.5/site-packages/futures-3.1.1.egg-info'...
Listing './lib/python3.5/site-packages/geo_pass'...
Compiling './lib/python3.5/site-packages/geo_pass/api.py'...
***   File "./lib/python3.5/site-packages/geo_pass/api.py", line 363
    w_intersects = filter(lambda (w, cuts): not buffer or nodes_in_buffer(cuts, buffer), all_intersects.items())
                                 ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/geo_pass-0.2.11.egg-info'...
Listing './lib/python3.5/site-packages/gunicorn'...
Listing './lib/python3.5/site-packages/gunicorn/app'...
Listing './lib/python3.5/site-packages/gunicorn/http'...
Listing './lib/python3.5/site-packages/gunicorn/instrument'...
Listing './lib/python3.5/site-packages/gunicorn/workers'...
Listing './lib/python3.5/site-packages/gunicorn-19.7.1.dist-info'...
Listing './lib/python3.5/site-packages/idna'...
Listing './lib/python3.5/site-packages/idna-2.6.dist-info'...
Listing './lib/python3.5/site-packages/itsdangerous-0.24.egg-info'...
Listing './lib/python3.5/site-packages/jinja2'...
Compiling './lib/python3.5/site-packages/jinja2/asyncfilters.py'...
***   File "./lib/python3.5/site-packages/jinja2/asyncfilters.py", line 24
    yield item
    ^
SyntaxError: 'yield' inside async function

Compiling './lib/python3.5/site-packages/jinja2/asyncsupport.py'...
***   File "./lib/python3.5/site-packages/jinja2/asyncsupport.py", line 35
    yield event
    ^
SyntaxError: 'yield' inside async function

Listing './lib/python3.5/site-packages/markupsafe'...
Listing './lib/python3.5/site-packages/overpy'...
Listing './lib/python3.5/site-packages/overpy-0.4.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pyproj'...
Listing './lib/python3.5/site-packages/pyproj/data'...
Listing './lib/python3.5/site-packages/pyproj-1.9.5.1.egg-info'...
Listing './lib/python3.5/site-packages/redis'...
Listing './lib/python3.5/site-packages/redis-2.10.6.dist-info'...
Listing './lib/python3.5/site-packages/redis_cache'...
Compiling './lib/python3.5/site-packages/redis_cache/rediscache.py'...
***   File "./lib/python3.5/site-packages/redis_cache/rediscache.py", line 93
    except RedisNoConnException, e:
                               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/redis_cache/test_rediscache.py'...
*** Sorry: TabError: inconsistent use of tabs and spaces in indentation (test_rediscache.py, line 193)
Listing './lib/python3.5/site-packages/redis_simple_cache-0.0.8.egg-info'...
Listing './lib/python3.5/site-packages/requests'...
Listing './lib/python3.5/site-packages/requests-2.18.4.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/shapely'...
Listing './lib/python3.5/site-packages/shapely/.libs'...
Listing './lib/python3.5/site-packages/shapely/algorithms'...
Listing './lib/python3.5/site-packages/shapely/examples'...
Listing './lib/python3.5/site-packages/shapely/geometry'...
Listing './lib/python3.5/site-packages/shapely/speedups'...
Listing './lib/python3.5/site-packages/shapely/vectorized'...
Listing './lib/python3.5/site-packages/tests'...
Listing './lib/python3.5/site-packages/tests/json'...
Listing './lib/python3.5/site-packages/tests/response'...
Listing './lib/python3.5/site-packages/tests/xml'...
Listing './lib/python3.5/site-packages/urllib3'...
Listing './lib/python3.5/site-packages/urllib3/contrib'...
Listing './lib/python3.5/site-packages/urllib3/contrib/_securetransport'...
Listing './lib/python3.5/site-packages/urllib3/packages'...
Listing './lib/python3.5/site-packages/urllib3/packages/backports'...
Listing './lib/python3.5/site-packages/urllib3/packages/ssl_match_hostname'...
Listing './lib/python3.5/site-packages/urllib3/util'...
Listing './lib/python3.5/site-packages/urllib3-1.22.dist-info'...
Listing './lib/python3.5/site-packages/werkzeug'...
Listing './lib/python3.5/site-packages/werkzeug/contrib'...
Listing './lib/python3.5/site-packages/werkzeug/debug'...
Listing './lib/python3.5/site-packages/werkzeug/debug/shared'...
Listing './share'...
Listing './share/python-wheels'...
